### PR TITLE
Sprint2 Development

### PR DESCRIPTION
**Pull request by cankurtur**
[MA-25] FetchImageView Component created.
[MA-0] PointAndDetailsContainerView created.
[MA-26] ImageCollectionViewWithPointAndDetailsContainerView created.
[MA-0] Created new network request for Now Playing Movies and also created response model, api path for this request.
[MA-23] New design made for now playing movies.
[MA-0] CardViewWithImageAndDetails component edited for accepting poster paths.
[MA-0] KingFisher added to project.

### CODE REVIEW

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/CollectionViewCell/CustomCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class CustomCollectionViewCell: UICollectionViewCell {
+    @IBOutlet private weak var imageView: FetchImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    func configure(image: FetchImageViewUIModel) {
```
Comment (batuhanabay) [inline movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/CollectionViewCell/CustomCollectionViewCell.swift]: You are not passing image as parameter, you are passing a uiModel into method, so we should give more accurate parameter names.

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        super.init(frame: frame)
+        commonInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    func commonInit() {
```
Comment (batuhanabay) [inline movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift]: It should be private

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        let view = nibInstantiate(autoResizingMask: [.flexibleHeight, .flexibleHeight])
+        view.frame = self.bounds
+        addSubview(view)
+        setupUICollectionView()
+    }
+
+    func setupUICollectionView() {
```
Comment (batuhanabay) [inline movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift]: It should be private

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift (hunk 2 above)
Comment (cankurtur) [reply movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift]: fixed.

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/CollectionViewCell/CustomCollectionViewCell.swift (hunk 1 above)
Comment (cankurtur) [reply movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/CollectionViewCell/CustomCollectionViewCell.swift]: fixed.

File: movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift (hunk 3 above)
Comment (cankurtur) [reply movieApp/movieApp/Components/ImageCollectionViewWithPointAndDetailsContainerView/ImageCollectionViewWithPointAndDetailsContainerView.swift]: fixed.

**Review by gidercagri — APPROVED** [review]

**Review by batuhanabay — APPROVED** [review]

**Pull request merged**